### DISCUSSION TITLE
logisim-evolution can be run on the Android platform.(3.8.0)

### DISCUSSION THREAD

**Discussion by oldcwj** [logisim-evolution/logisim-evolution#2066] [General]
logisim-evolution can be run on the Android platform. There is an app called [jre4android.](https://play.google.com/store/apps/details?id=com.coobbi.jre) (java 17) The running effect is as follows
![logisim-evolution](https://github.com/user-attachments/assets/2d4e19e8-a419-4b03-b46e-a82393806d34)

**Comment by oldcwj** [comment]
logisim-evolution-3.8.0-all.jar can run on it

**Comment by oldcwj** [comment]
jre4android pro(jdk version is 21) support 3.9.0

**Comment by star886** [comment]
Or use FCL[https://github.com/FCL-Team/FoldCraftLauncher](https://github.com/FCL-Team/FoldCraftLauncher),although it's just a mobile minecraft launcher
![Screenshot_2025-02-27-14-50-22-12_c8a66bc90793a73eafa5773962b202df](https://github.com/user-attachments/assets/6dd7cf84-31c3-4326-bb09-937cd37e8584)

**Reply by Nico343** [reply]
Commenting to fix the link:

[https://github.com/logisim-evolution/logisim-evolution/discussions/url](https://github.com/FCL-Team/FoldCraftLauncher)

**Reply by maehne** [reply]
I fixed the original link in the above [message](https://github.com/logisim-evolution/logisim-evolution/discussions/2066#discussioncomment-12334877).
